junit5/src/main/java/com/linecorp/armeria/testing/junit5/server/ServerExtension.java

```diff
@@ -66,6 +71,7 @@ public void configure(ServerBuilder sb) throws Exception {
     public void before(ExtensionContext context) throws Exception {
         try {
             delegate.before();
+            contextCaptor.clear();
```

Comment:
Shouldn't we clear in `beforeEach`?

Reply:
> Shouldn't we clear in `beforeEach`?

There is no `beforeEach` method in `ServerExtension` class 😄

Reply:
Then, how about overriding it?
```java
    @Override
    public final void beforeEach(ExtensionContext context) throws Exception {
        if (runForEachTest()) {
            before(context);
        }
        contextCaptor.clear();
    }
```

Reply:
> Then, how about overriding it?
> 
> ```java
>     @Override
>     public final void beforeEach(ExtensionContext context) throws Exception {
>         if (runForEachTest()) {
>             before(context);
>         }
>         contextCaptor.clear();
>     }
> ```

It its `final`, do you want me to remove `final`?

Reply:
Yes, I think we can do that. 😄

Reply:
Hm, let me think a bit more to think about it.

Reply:
I can't think of any good way except overriding the method.
Do you have any thoughts? @trustin